scripts/gtk-bundle-from-msys2.sh

```diff
@@ -1,43 +1,51 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
why?  I didn't test, but didn't spot any obvious bashisms, are there any (really) required?

Reply:
There are a couple of `$(command...)` in there even though I might added them after I set it to bash :+1: 
The reason is this way we don't need to care whether to use bashisms or not. And shell portability isn't an issue for this particular script. It is used in a Docker container where bash is always available (as defined by Dockerfile) or it is used on MSYS2 where bash is an essential package.
So we just make our own life easier. I'm a big fan of portable scripts but here it would be simply overkill.

Reply:
`$(…)` is no bashism :)

But OK, your answer fair enough, color me convinced.